llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -7764,7 +7550,9 @@ VectorizationFactor LoopVectorizationPlanner::computeBestVF() {
   }
 
   for (auto &P : VPlans) {
-    for (ElementCount VF : P->vectorFactors()) {
+    SmallVector<ElementCount, 1> VFs(P->vectorFactors());
```

Comment:
Are we expecting to only have one VF? Should we just let the template default choose the small size
```suggestion
    SmallVector<ElementCount> VFs(P->vectorFactors());
```